massgov/mayflower
# DP-10462 add toc to multiple

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Add the sticky table of contents to multiple new types of content.

## Related Issue / Ticket

- [DP-10774](https://jira.mass.gov/browse/DP-10774)

## Steps to Test


1. Checkout the branch and build Mayflower locally
1. Confirm that the sticky TOC appears on the following pages and functions correctly at different breakpoints and with different browsers:
    - [Templates -> Content Types -> Court Rules](http://localhost:3000/patterns/04-templates-01-content-types-court-rules/04-templates-01-content-types-court-rules.html)
    - [Templates -> Content Types -> Policy Advisory](http://localhost:3000/patterns/04-templates-01-content-types-policy-advisory/04-templates-01-content-types-policy-advisory.html)
    - [Pages -> Board Decision](http://localhost:3000/patterns/05-pages-board-decision/05-pages-board-decision.html)
    - [Pages -> Executive Order](http://localhost:3000/patterns/05-pages-executive-order/05-pages-executive-order.html)
    - [Pages -> Guide](http://localhost:3000/patterns/05-pages-guide/05-pages-guide.html)
    - [Pages -> Information Details](http://localhost:3000/patterns/05-pages-information-details/05-pages-information-details.html)
    - [Pages -> Information Details Precontent Media](http://localhost:3000/patterns/05-pages-information-details-preContent-media/05-pages-information-details-preContent-media.html)
    - [Pages -> Information Details Table of Contents](http://localhost:3000/patterns/05-pages-information-details-table-of-contents/05-pages-information-details-table-of-contents.html)
    - [Pages -> Listing Links](http://localhost:3000/patterns/05-pages-listing-links/05-pages-listing-links.html)
    - [Pages -> Policy Advisory Directive](http://localhost:3000/patterns/05-pages-policy-advisory-directive/05-pages-policy-advisory-directive.html)
    - [Pages -> Regulation](http://localhost:3000/patterns/05-pages-regulation/05-pages-regulation.html)
    - [Pages -> Rules of Court](http://localhost:3000/patterns/05-pages-rules-of-court/05-pages-rules-of-court.html)

*Note this cannot be tested in Drupal with just updating Mayflower. Drupal branch will be available shortly.*


## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
